cabal-install/src/Distribution/Client/Dependency.hs

```diff
@@ -13,6 +13,7 @@
 -- Top level interface to dependency resolution.
 -----------------------------------------------------------------------------
 module Distribution.Client.Dependency (
+    DepResolverParams,
```

Comment:
Which section should this fall under?

Reply:
I bet @grayjay would know...

Reply:
The top section looks like a good match to me, since DepResolverParams is one of the top level types for running the dependency solver.

Reply:
done